invenio_app_rdm/records_ui/views/records.py

```diff
@@ -195,8 +195,12 @@ def record_detail(
     if record is not None and emitter is not None:
         emitter(current_app, record=record._record, via_api=False)
 
-    # NOTE: this should maybe be an expandable field instead
-    record_owner = record._record.parent.access.owner.resolve()
+    record_owner = (
+        record_ui.get("expanded", {})
+        .get("parent", {})
+        .get("access", {})
+        .get("owned_by", {})
```

Comment:
```suggestion
        .get("owned_by")
```
and then I think you don't have to check for `record_owner != {}` below